sections/semantics-forms.include

```diff
@@ -6336,6 +6336,33 @@ You cannot submit this form when the field is incorrect.</samp></pre>
 
   </div>
 
+<h6 id="the-autocapitalize-attribute">The <code>autocapitalize</code> attribute</h6>
+
+    The <dfn element-attr for="input"><code>autocapitalize</code></dfn> attribute provides
+    information to the user agent on the text capitalization a user's keyboard should
+    use on text fields.
+
+    It currently provides three different capitalization methods; <code>sentences</code>,
+    <code>words</code>, and <code>characters</code>.
+
+    The <code>autocapitalize</code> attribute is a suggestion. The user must be able alter the
+    text they input.
+
+    <div class="example">
+      Here are examples of text entry fields which uses the <code>autocapitalize</code> attribute:
+
+      <pre highlight="html">
+      &lt;label&gt;Sentence case. The first character after a period will be capitalized.&lt;/label&gt;
+      &lt;input type="text" autocapitalize="sentences"&gt;
+
+      &lt;label&gt;Word Case. This Capitalizes The First Letter Of Every Word&lt;/label&gt;
+      &lt;input type="text" autocapitalize="words"&gt;
+
+      &lt;label&gt;CHARACTER CASE. EVERY CHARACTER IS CAPITALIZED.&lt;/label&gt;
+      &lt;input type="text" autocapitalize="characters"&gt;
+
+    </div>
+
```

Comment:
Please note the addition in the changes section (changes.include in the sections directory).

Reply:
New version updates the changes.include

Reply:
Thanks